### DISCUSSION TITLE
How can I configure an application to run on Mac OSX (without allowing third party applications)

### DISCUSSION THREAD

**Discussion by cienciascontic** [SafeExamBrowser/seb-mac#114] [Q&A]
Hi!
I want my students to be able to run Jamovi on their Macs in this way:
![image](https://user-images.githubusercontent.com/6078186/129953841-c7083584-1b44-4f26-865f-db4ef34ebeb8.png)
It works fine on Windows machines but not on Mac OSX. What am I doing wrong? Or is it just that on Mac the only way to do that is by allowing third party applications???
If that is true, why does SEB configuration tab has the option to choose the OS (Win or OSX)?
Thanks for your help.
Regards,

Cristián

**Comment by danschlet** [comment]
As the [SEB for macOS manual](https://safeexambrowser.org/macosx/mac_usermanual_en.html#PermittedProcessesSection) states:

Using permitted processes is not supported in SEB for macOS. You can define and edit permitted processes for SEB for Windows here.

The feature will be supported in future SEB for macOS versions, as Apple now allows using additional applications with the AAC Assessment Mode (built-in macOS kiosk/lockdown mode), but only with macOS 12 Monterey and newer. It won't be able to prevent that such additional applications access the macOS file system though.

With the standard SEB for Mac kiosk mode it's technologically not possible to allow applications to run along in a secure way. You could use a virtual desktop environment which allows to access a virtual Windows desktop in a full screen browser window (like VMware Horizon or cloud solutions by Amazon or Microsoft).
